# -*- coding:utf-8 -*-
# $ catkin_make -DCATKIN_WHITELIST_PACKAGES="copley"

# Header
Header header
# Drive
copley/motors_msg current
copley/motors_msg velocity
copley/motors_msg position
